content/docs/run-api-server/ingestion.mdx

```diff
@@ -111,15 +111,15 @@ In some (albeit rare) cases, it can be convenient to (re)ingest the full Stellar
 The following instructions assume the reingestion is done on AWS. However, they should be applicable to any other environment with equivalent capacity. In the same way, the instructions can be adapted to reingest only specific parts of the history.
 
 ### Prerequisites
-Before we begin, we make some environment assumptions:
+Before we begin, we make some assumptions around the environment required. Please refer to the [Prerequisites](./prerequisites.mdx) section for the current HW requirements to run Horizon reingestion (for either historical catch up) or real-time ingestion (for staying in synch with the ledger). A few things to keep in mind:
+ 
+  1. For reingestion, the more parallel workers are provisioned to speed up the process, the larger the machine size is required in terms of RAM, CPU, IOPS and disk size. The size of the RAM per worker also increases over time (14GB RAM / worker as of mid 2022) due to the growth of the ledger. HW spec can be downsized once Re-ingestion is completed. 
 
-  1. An instance with 32 CPU cores, 64 GB of RAM and at least 200 GB of disk capacity from which to run Horizon (for example AWS `m5.8xlarge`). This is needed to fit 20 Horizon parallel workers (each with their own Captive Core instance). Each Core instance can take up to 3GB of RAM and a full CPU core (more on why 20 workers below). If the number of workers is increased, you may need a larger machine.
+  2. [Horizon](./installing.mdx) version 2.1x or newer installed on the machine from (1).
 
-  2. [Horizon](./installing.mdx) version 1.6.0 or newer (ideally, 2.x) installed on the machine from (1).
+  3. [Core](https://github.com/stellar/stellar-core) version 18.5 or newer  installed on the machine from (1).
```

Comment:
Should we just stop mentioning a version here and just say "the latest version of stellar-core" and link to https://github.com/stellar/stellar-core/releases/latest?

Reply:
And likewise for Horizon?

Reply:
Yes good idea!